Breanna/fade_Code.ino

```diff
@@ -0,0 +1,30 @@
+
+
+//sort of like a key that allows you to use just the 'int' name insted of the number. 
```

Comment:
I really like the way you write your comments. They are more specific and I need to take note!